rfcs/0017-tx-valid-since/0017-tx-valid-since.md

```diff
@@ -15,97 +15,246 @@ This RFC suggests adding a new consensus rule to prevent a cell to be spent befo
 
 ## Summary 
 
-Transaction adds a new `u64` type field `valid_since`, which prevents the transaction to be mined before a certain point, the highest bit of `valid_since` is `type flag`, the other 63 bits represent the actual value `V`, the consensus to validate this field described as follow:
+Transaction input adds a new `u64` type field `valid_since`, which prevents the transaction to be mined before an absolute or relative time.
```

Comment:
`u64` better replace with **`u64` (unsigned 64 bit integer)** for none-rust reader.